ibm_cloud_sdk_core/base_service.py

```diff
@@ -136,6 +159,14 @@ def configure_service(self, service_name: str) -> None:
             )
         if config.get('ENABLE_GZIP') is not None:
             self.set_enable_gzip_compression(config.get('ENABLE_GZIP') == 'True')
+        if config.get('MAX_RETRIES'):
+            self.set_max_retries(
+                int(config.get('MAX_RETRIES'))
```

Comment:
With the new `enable_retries(...)` method, if the ENABLE_RETRIES property is set to true, then we would retrieve the MAX_RETRIES and RETRY_INTERVAL properties, then call `enable_retries()` instead of having individual setters for max_retries and retry_interval.